jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -5354,4 +5354,45 @@ public IntStream codePoints() {
 		return value;
 	}
 /*[ENDIF] Java10 */	
+
+/*[IF Java11]*/
+	/**
+	 * Returns a string object containing the character (Unicode code point)
+	 * specified.
+	 * 
+	 * @param codePoint
+	 *            a Unicode code point.
+	 * @return a string containing the character (Unicode code point) supplied.
+	 * @throws IllegalArgumentException
+	 *             if the codePoint is not a valid Unicode code point.
+	 * @since 11
+	 */
+	static String valueOfCodePoint(int codePoint) {
+		String string;
+		// Save less often check (codePoint > Character.MAX_CODE_POINT) last
+		if (codePoint < Character.MIN_CODE_POINT) {
+			/*[MSG "K0800", "Invalid Unicode code point"]*/
+			throw new IllegalArgumentException(com.ibm.oti.util.Msg.getString("K0800"));	//$NON-NLS-1$
```

Comment:
Can you include the string or the the code point in the exception?  It will make it much easier to debug one of these issues.

Reply:
Yup, now an exception looks like:
```
FAILED: testValueOfCodePoint
java.lang.AssertionError: Unicode code point mismatch at 0 expected [1] but found [0]
	at org.testng.Assert.fail(Assert.java:96)
	at org.testng.Assert.failNotEquals(Assert.java:776)
	at org.testng.Assert.assertEqualsImpl(Assert.java:137)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:652)
	at org.openj9.test.java_lang.Test_String.testValueOfCodePoint(Test_String.java:49)
```